logstash-core/src/main/java/org/logstash/Valuefier.java

```diff
@@ -22,6 +22,7 @@
 
 import java.math.BigDecimal;
 import java.math.BigInteger;
+import java.time.*;
```

Comment:
nitpick: we tend to avoid wildcard imports, it'd be preferable to explicitly include the java Date classes here